nss-day-cohort-30/bangazon-site-mighty-mahi-mahi
# Product rating

Fixes:
- Users can now rate orders from completed orders
- Users listed products now reflect an avg rating

Changes proposed in this request & reasons behind organizational or architectural decisions:
- Created new database join table to capture product ratings

Steps to test:
-Start Bangazon
- check your listed products
- buy one of your own products
- go to the order in your order history, rate the product
- go back to your listed products to see the average rating

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-n/a

Commit message:
- users listed products show average ratings if they have been rated in completed  orders
--

Link to feature ticket:
-https://github.com/nss-day-cohort-30/bangazon-site-mighty-mahi-mahi/issues/22

@mighty-mahi-mahi
